### DISCUSSION TITLE
does github have a place where you can report scams?

### DISCUSSION THREAD

**Discussion by zevxis** [community/community#147374] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i fell victim to crypto thievery tonight by a (purported) user on github, 'Thorazine361'. i couldn't find any specific area to report it. he answered my message for help in recovering one of my wallets referring me to the URL 'https://livechatsupports.app/wallets.html#'. if anyone could help, i'd appreciate...regards, z

**Comment by shinybrightstar** [comment]
Hi @zevxis , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
